src/item.cpp

```diff
@@ -1870,9 +1871,11 @@ nc_color item::color_in_inventory() const
         switch( rating ) {
             case EDIBLE:
             case TOO_FULL:
-                if( is_going_bad() ) {
+                if( preserves ) {
+                    // Nothing, canned food won't rot
```

Comment:
Which sounds like what I first assumed the bug was, I assumed it was a positive change before realizing that it also applied to containers that didn't preserve rot.

Then again, if we made cans need a can opener and made open cans no longer preserve food, I guess that technically would render it a moot point. XP